### DISCUSSION TITLE
Multiple sequences responding exclusively

### DISCUSSION THREAD

**Discussion by warp929** [evert-arias/EasyButton#63] [Q&A]
Hello. I have tried a few different ways of getting 1,2 and 3 button presses to respond exclusively.  Meaning you only get one of the results.  Effectively, a single button press would need to wait for a timeout before it shows up as a single button press.  My latest test code is initializing 3 sequences:
` button.onSequence(1, 2000, onSequenceMatched1);

button.onSequence(2, 2000, onSequenceMatched2);

button.onSequence(3, 2000, onSequenceMatched3);
`

Based on your Sequence example, a triple button press results in:
_single pressed
single pressed
double pressed
single pressed
triple pressed_

I would like the response to only be "triple pressed"

Is there a way to accomplish this with the EasyButton library?
Here is somebody else asking the same question:
https://forum.arduino.cc/t/easybutton-double-click-problem/639430/2

**Comment by evert-arias** [comment]
@warp929 I will check this out.
